workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -424,11 +407,20 @@ private void SetupDispatcherHandlers()
 
         private static class Errors
         {
-            public const string DuplicateAdditionOfEntity =
-                "Tried to add an entity but there is already an entity associated with that EntityId.";
+            public const string EntityAlreadyExistsError =
+                "Received an AddEntityOp with Spatial entity ID {0}, but an entity with that EntityId already exists.";
+
+            public const string EntityNotFoundForDeleteError =
+                "Received a DeleteEntityOp with Spatial entity ID {0}, but an entity with that EntityId could not be found."
+                + "This could be caused by deleting SpatialOS entities locally. "
+                + "Use a DeleteEntity command to delete entities instead.";
```

Comment:
A reasonable user question would be what is "a DeleteEntity command" - more clarity may be useful here.